code/datums/shuttles.dm

```diff
@@ -5,6 +5,10 @@
 	var/port_id
 	var/shuttle_id
 
+	// If counter is non-0, it's incremented and added to the shuttle_id when
+	// the template is imported. Used for pods eg, pod1, pod2, pod3
+	var/counter = 0
```

Comment:
If that's the case, maybe it should be FALSE here and TRUE on the things that activate it, since those amount to 0 and 1